app/workers/migrate_user_settings_worker.rb

```diff
@@ -0,0 +1,82 @@
+class MigrateUserSettingsWorker
```

Comment:
Given that this worker is only going to be used as support of data update scripts, maybe we should inline it in the same file so it's self contained

Reply:
+ 1 on this.

Reply:
@rhymes I _love_ this idea. Would Rails autoload the constant even in the `lib/data_update_scripts` directory, though? If not, do you know of a good way to get Sidekiq to load it without hardcoding it into a `require` statement?

Reply:
Nope, other than loading the class at runtime I don't know